src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/legalrepresentative/LegalRepresentativeEmailNotificationPersonalisation.java

```diff
@@ -1,28 +1,41 @@
-package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.legalrepresentative;
-
-import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.CHANGE_ORGANISATION_REQUEST_FIELD;
-import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.LEGAL_REPRESENTATIVE_EMAIL_ADDRESS;
-
-import java.util.Collections;
-import java.util.Set;
-import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
-import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.field.ChangeOrganisationRequest;
-import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
-
-public interface LegalRepresentativeEmailNotificationPersonalisation extends EmailNotificationPersonalisation {
-
-    @Override
-    default Set<String> getRecipientsList(AsylumCase asylumCase) {
-
-        if (asylumCase.read(CHANGE_ORGANISATION_REQUEST_FIELD, ChangeOrganisationRequest.class)
-            .map(it -> it.getCaseRoleId() == null)
-            .orElse(false)) {
-
-            return Collections.emptySet();
-        } else {
-            return Collections.singleton(asylumCase
-                .read(LEGAL_REPRESENTATIVE_EMAIL_ADDRESS, String.class)
-                .orElseThrow(() -> new IllegalStateException("legalRepresentativeEmailAddress is not present")));
-        }
-    }
-}
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.legalrepresentative;
+
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.CHANGE_ORGANISATION_REQUEST_FIELD;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.JOURNEY_TYPE;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.LEGAL_REPRESENTATIVE_EMAIL_ADDRESS;
+
+import java.util.Collections;
+import java.util.Objects;
+import java.util.Set;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.JourneyType;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.field.ChangeOrganisationRequest;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
+
+public interface LegalRepresentativeEmailNotificationPersonalisation extends EmailNotificationPersonalisation {
+
+    @Override
+    default Set<String> getRecipientsList(AsylumCase asylumCase) {
+
+        if (asylumCase.read(JOURNEY_TYPE, JourneyType.class)
+            .map(type -> Objects.equals(type.getValue(), JourneyType.AIP.getValue()))
+            .orElse(false)) {
+
+            return Collections.emptySet();
+        }
+
+        if (asylumCase.read(CHANGE_ORGANISATION_REQUEST_FIELD, ChangeOrganisationRequest.class)
+            .map(it -> it.getCaseRoleId() == null)
+            .orElse(false)) {
+
+
+

```

Comment:
Unnecessary whitespace.

Also: the whole file looks modified instead of specific lines. Did the Line Feed convention change from LF to CRLF? If so, please re-instate LF.